azure/activity_logs_monitoring/index.js

```diff
@@ -135,22 +135,31 @@ function extractResourceId(record) {
     metadata = { tags: [], source: '' };
     if (
         record.resourceId === undefined ||
-        typeof record.resourceId !== 'string' ||
-        !record.resourceId.toLowerCase().startsWith('/subscriptions/')
+        typeof record.resourceId !== 'string'
     ) {
         return metadata;
     }
-    var resourceId = record.resourceId.toLowerCase().split('/');
-    if (resourceId.length > 2) {
-        metadata.tags.push('subscription_id:' + resourceId[2]);
+    else if(record.resourceId.toLowerCase().startsWith('/subscriptions/')){
+      var resourceId = record.resourceId.toLowerCase().split('/');
+      if (resourceId.length > 2) {
+          metadata.tags.push('subscription_id:' + resourceId[2]);
+      }
+      if (resourceId.length > 4) {
+          metadata.tags.push('resource_group:' + resourceId[4]);
+      }
+      if (resourceId.length > 6 && resourceId[6]) {
+          metadata.source = resourceId[6].replace('microsoft.', 'azure.');
+      }
+      return metadata;
     }
-    if (resourceId.length > 4) {
-        metadata.tags.push('resource_group:' + resourceId[4]);
+    else if(record.resourceId.toLowerCase().startsWith('/tenants/')){
+      var resourceId = record.resourceId.toLowerCase().split('/');
+      if (resourceId.length > 2 && resourceId[4]) {
+          metadata.tags.push('tenant:' + resourceId[2]);
+          metadata.source = resourceId[4].replace('microsoft.', 'azure.').replace('aadiam', 'active_directory');
```

Comment:
@anshumgargdd we'll want to update this to remove the '_' like we talked about. This will make it match the format of all the other azure source.

Updatted format should be: 'source:azure.activedirectory'

Reply:
Done.